packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -53,7 +53,7 @@ type TAddressDetails = {
 };
 
 type TAutoComplete = {
-    value: boolean;
+    value: string;
```

Comment:
Do we still need the value here?

Reply:
Yes we do I saw the use of it here https://github.com/binary-com/deriv-app/pull/14086/files#diff-19d2edcfca855845968113d4d5931538befdf96d97346b319a54c1157c0203bcR91 can ignore thanks

Reply:
yes , it's fixing the TS error for field here
![Uploading image.png…]()